Packs/CortexXDR/Playbooks/PaloAltoNetworks_Cortex_XDR_Incident_Sync.yml

```diff
@@ -230,7 +230,5 @@ view: |-
   }
 inputs: []
```

Comment:
do we need this from 6.0?

Reply:
Good question - I think not - @anara123?

Reply:
We don't